server/container_create_linux.go

```diff
@@ -33,9 +33,14 @@ import (
 	pb "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
 )
 
-// minMemoryLimit is the minimum memory that must be set for a container.
-// A lower value would result in the container failing to start.
-const minMemoryLimit = 4194304
+const (
+	// minMemoryLimit is the minimum memory that must be set for a container.
+	// A lower value would result in the container failing to start.
+	minMemoryLimit = 4194304
+	// defaultInitLocation describes where an init binary should be
+	// mounted into the container
+	defaultInitLocation = "/dev/init"
```

Comment:
I think /var/ may be more suitable. cc: @rhatdan

Reply:
Re-reading the FHS, `/sbin/init` is the better location for this.

Reply:
hm, I was conforming with podman on this one. Also, for the majority of the mounts we use we mount in /dev, so isn't it idiomatic to do that here as well?

Reply:
Seems wonky to me to mount a binary under /dev. Let's hear what @rhatdan thinks.

Reply:
Yes /dev/init does seem strange but I believe the location was created based on what Docker was doing.

Reply:
so what's our consensus?

Reply:
I vote something under `/sbin` but not overwrite `init` so maybe `/sbin/crioinit`.

Reply:
SGTM, @rhatdan ?